.github/workflows/ci.yml

```diff
@@ -17,10 +17,19 @@ jobs:
       fail-fast: false
       matrix:
         ruby: [ "2.0.0", 2.1, 2.3, 2.4, 2.5, 2.6, 2.7, "3.0", 3.1, 3.2, 3.3, jruby-9.1, jruby-9.2, jruby-9.3, jruby-9.4 ]
+        latest_commonmarker: ["", "yes"]
```

Comment:
With the `.ci.gemfile` changes recommended, all changes in this file can be removed.

These changes double the number of CI jobs, which I don't want to do.  It's fine to test commonmarker 1.0+ on Ruby 3.1+, and commonmarker <1.0 on Ruby <3.1.